src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -2404,8 +2405,29 @@ namespace InternalHeatGains {
                             ErrorsFound = true;
                         }
                         thisZoneOthEq.otherEquipFuelTypeString = FuelTypeString; // Save for output variable setup later
-                        state.dataHeatBal->Zone(zoneNum).otherEquipFuelTypes.emplace_back(FuelTypeString);
-                        state.dataHeatBal->space(spaceNum).otherEquipFuelTypes.emplace_back(FuelTypeString);
+                        // Build list of fuel types used in each zone and space (excluding None and Water)
```

Comment:
Changes here towards addressing the other fuel type warnings (mainly now the list is of *unique* fuel types where it wasn't before).